### DISCUSSION TITLE
Github account got flagged as spam, please help

### DISCUSSION THREAD

**Discussion by jimmyzzxhlh2** [community/community#23775] [New to GitHub]
<p>Hi,</p>
<p>Can someone kindly help with my github account <code>jimmyzzxhlh</code> which got flagged incorrectly as spam? I have to register another account for making this post since the original account cannot be authorized to Github community platform. I used the account for development in my company and my teammates cannot see my pull requests/reviews any more, which is super annoying as we are committing to release our projects recently.</p>
<p>Related github support request is <span class="hashtag">#718828</span>. Thank you and please don’t hesitate to let me know if you have any questions or concerns.</p>
<p>Regards,<br>
-Zhongxia</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>I’ve had a look it I think this has been sorted out for you. Let us know if you’re still having trouble.</p>

**Reply by SahilPanigrahi** [reply]
Hi,
My account also got flagged as spam, can you help?
I am not a spammer, but my account was being marked as spam. I also don't know need to do what to unflag the spam on my account.

**Reply by SahilPanigrahi** [reply]
https://github.com/SahilPanigrahi

**Reply by hacanand** [reply]
please took into my account also it was flagged spam without any reason

**Reply by M-Shariq-546** [reply]
Hello @canuckjacq My account is also labelled as spammy please help me as well for the solution of this issue

**Reply by JahongirDevelop** [reply]
Hello @canuckjacq My account is also labelled as spammy please help me as well for the solution of this issue

**Reply by vitytradez** [reply]
Hey @canuckjacq I appreciate that you are here to help some people out. I have been flagged for about a month now, and can't log in to some third party applications via github that contain very important data.

**Reply by janu3605** [reply]
Hello, 
my account too has been flagged as spam for a while now. I had 2 factor authentication setup after that, but I still haven't been able to sort it out. could you please check.

https://github.com/janu3605

**Reply by chenbowu** [reply]
Hey @canuckjacq,

I am trying to clone my repository from local machine. But i am getting the follow exception that "You are maked as spam, and therefore cannot authorize a third party application.". Can you please solve this challenge please dont do anything.

**Reply by sean-sc-lee-appier** [reply]
Hi,
my account has been flagged as spam for a while, and I had 2 factor authentication setup now, but my account is still flagged. Could you please check? thanks a lot.

https://github.com/sean-lee-appier

**Comment by devtmj1123** [comment]
Hi,
My account also got flagged as spam, can anyone help?
I am not a spammer, but my account was being marked as spam, so I tried to remove my primary and backup mail and it didn't work. I also don't know need to do what to unflag the spam on my account.

**Comment by sharjeel0288** [comment]
Hi,
My account also got flagged as spam, can you help?
I am not a spammer, but my account was being marked as spam. I also don't know need to do what to unflag the spam on my account.

**Comment by byronhenry349** [comment]
Hello @canuckjacq , I created GitHub account to request a feature for SimpleX Chat (https://github.com/simplex-chat/simplex-chat/) and opened an issue (https://github.com/simplex-chat/simplex-chat/issues/3780) but it's not visible after I log out

I opened GitHub Support page and it says that my account has been flagged, but I'm not able to open ticket because "captcha result is invalid" i.e. it shows green tick but later shows this error:

> Sorry, the captcha result was invalid. Please visit https://docs.github.com/articles/troubleshooting-connectivity-problems/#troubleshooting-the-captcha for troubleshooting information.

As a proud Tor user who wants to anonymously contribute to open-source software I feel discriminated against by GitHub

Can you please fix it for me? Thank you in advance

**Comment by hacanand** [comment]
my account is flagged as spam so please review this and fix it

**Comment by AmineRACHID** [comment]
Hi,
My account also got flagged as spam, can anyone help?
I am not a spammer, but my account was being marked as spam, so I tried to remove my primary and backup mail and it didn't work. I also don't know need to do what to unflag the spam on my account.

**Comment by sai-ganesh-03** [comment]
My Account is flagged as spam. I'm not a spammer. Please help!!

**Comment by yeahjjyy** [comment]
My Account is flagged as spam. I'm not a spammer. Please help!! 
My organization requires a lot of actions

**Comment by hazzmatt1** [comment]
I have been flagged as spam as well. How do we resolve this?

**Comment by chenbowu** [comment]
Hi Github Team,

I am trying to clone my repository from local machine. But i am getting the follow exception that "You are maked as spam, and therefore cannot authorize a third party application.". Can you please solve this challenge please dont do anything.

**Comment by lamechy** [comment]
My account was also flagged as spam. Please help, i raised a request number #4137761

**Comment by alegd** [comment]
My account was also flagged as spam, not cool, all my projects are here. I open this ticket https://support.github.com/ticket/4243676
